src/earn/EarnEnterAmount.tsx

```diff
@@ -376,57 +381,81 @@ function EarnEnterAmount({ route }: Props) {
         <EarnDepositBottomSheet
           forwardedRef={reviewBottomSheetRef}
           preparedTransaction={prepareTransactionsResult}
-          amount={tokenAmount.toString()}
+          amount={tokenAmount}
           tokenId={token.tokenId}
+          apy={apy}
+          token={token}
           networkId={token.networkId}
         />
       )}
     </SafeAreaView>
   )
 }
 
-function EarnProceed({
+export function EarnApyAndAmount({
```

Comment:
can this component just use the hook and selector for getting the APY and localCurrencySymbol? instead of getting it at a higher level component and passing it around as props to a bunch of them